app/services/paypal_service/api/payments.rb

```diff
@@ -37,7 +36,7 @@ def request(community_id, create_payment, force_sync: true)
       @lookup.with_active_account(
         community_id, create_payment[:merchant_id]
       ) do |m_acc|
-        if (use_async?(force_sync))
+        if !force_sync
```

Comment:
So what this changes is that is removes the ability to run PayPal payments "synchronously", which in this case means, run payments without Delayed Jobs. Synchronized execution was previously then only way to use FakePal, because FakePal didn't have a way to share memory between app process and the worker process. Now that the implementation uses the file-based storage, FakePal can share state with two processes and thus FakePal can be run in asynchronous manner, the same way as the real implementation.

I hate the work sync/async in this context! They are ambiguous: In the context of the transaction layer, it means synchronous as in Braintree (you just enter the CC details in the payment view and that's it) or asynchronous as in PayPal (you're redirected to 3rd-party site, payment will be finalized when the user comes back). In PayPal, the word sync/async means the way how the code is executed: synchronously in app-process or asynchronously in worker process.